ddpui/ddpairbyte/airbytehelpers.py

```diff
@@ -966,7 +966,7 @@ def fetch_and_update_org_schema_changes(org: Org, connection_id: str):
     except Exception as err:
         return (
             None,
-            f"Something went wrong fetching schema change (catalog) for connection {connection_id}: {err}",
+            f"Something went wrong fetching schema change (catalog) for {org.slug} connection {connection_id}: {err}",
```

Comment:
we should figure & get some consistency in our logging code. some places we use f-strings while other places we use `%s` with format strings